src/components/CookiesDialog.vue

```diff
@@ -0,0 +1,118 @@
+<!-- eslint-disable vue-i18n/no-raw-text -->
+<template>
+  <div
+    class="cookies-dialog"
+    @click.stop
+  >
+    <ButtonPlain
+      class="button-cancel"
+      @click="$emit('click', $event)"
```

Comment:
```suggestion
      @click="$emit('close', $event)"
```
or something more suitable than `click`